src/main/scala/com/itechart/ny_accidents/Application.scala

```diff
@@ -53,41 +54,41 @@ object Application extends App {
 
 
   val report = injector.getInstance(classOf[Reports])
-  val dayOfWeekReport = report.generateReportForTupleRDD[(String, Int, Double)](dayOfWeek)
+  val dayOfWeekReport = report.generateReportForTupleRDD[(String, Int, Double)](dayOfWeek, DAY_OF_WEEK_REPORT_HEADER)
```

Comment:
Could you clarify the purpose of (dayOfWeek, DAY_OF_WEEK_REPORT_HEADER)?
or
(hourOfDay, HOUR_OF_DAY_REPORT_HEADER)
or
(districtsPercentage, DISTRICT_REPORT_HEADER)

Sounds like there two arguments are coupled somehow, which is an indication of bad design

Reply:
first argument is data, and second is a csv header for this data.

Reply:
ok, makes sense